python/lsst/dax/webservcommon/auth.py

```diff
@@ -0,0 +1,72 @@
+import json
+from functools import wraps
+
+import jwt
+import requests
+from flask import request, g
+from jwt.algorithms import RSAAlgorithm
+
+ALGORITHM = "RS256"
+
+
+class DaxAuthException(Exception):
+    pass
+
+
+class JwtAuth:
+
+    def __init__(self, key, issuer=None, audience=None, options=None):
+        """
+        JwtAuth is a thin wrapper around the PyJWT project. Parameters
+        are best documented in that project.
+        :param key: Public key to verify RS256 signatures with
+        :param issuer: Domain of issuer, if applicable
+        :param audience: Audience of token, if applicable. If this is
+        false, then the check is turned off
+        :param options: verification options to pass to decode
```

Comment:
I think we are supposed to use numpydoc style for docstring, or doesn't it apply do DAX services?